main/reference/vstorage-ref.md

```diff
@@ -368,6 +420,39 @@ The `published.wallet.${address}` key has wallet's last update. For example:
           },
         ],
       },
-      updated: 'balance',
-    }
+      brand: Object @Alleged: Zoe Invitation brand#board0074 {},
+    },
+  ],
+}
+```
+
+The `published.wallet.${address}` key has wallet's last update. For example:
+
+```js
+{
+  currentAmount: {
+    brand: Object @Alleged: Zoe Invitation brand#board0074 {},
+    value: [
+      {
+        description: 'Voter0',
+        handle: Object @Alleged: InvitationHandle#null {},
+        installation: Object @Alleged: BundleIDInstallation#board00613 {},
+        instance: Object @Alleged: InstanceHandle#null {},
+      },
+      {
+        description: 'charter member invitation',
+        handle: Object @Alleged: InvitationHandle#null {},
+        installation: Object @Alleged: BundleIDInstallation#board01679 {},
+        instance: Object @Alleged: InstanceHandle#null {},
+      },
+    ],
+  },
+  updated: 'balance',
+}
 ```
+
+See also:
+
+- [Querying VStorage](/guides/getting-started/contract-rpc#querying-vstorage)
```

Comment:
https://docs.agoric.com/guides/getting-started/ui-tutorial/querying-vstorage.html should probably go before `/getting-started/contract-rpc` in this see also list